api/MouseEvent.json

```
@@ -1303,13 +1313,13 @@
               "version_added": "3.1"
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "≤3"
```

If we have confidence in 3.1 as the Safari version I think we could set iOS to 2, it can't be wrong by much:

```suggestion
              "version_added": "2"
```